solr/solrj/src/test/org/apache/solr/common/util/TestRetryUtil.java

```diff
@@ -68,4 +69,64 @@ public void testRetryOnThrowable() throws Throwable {
     assertTrue(executes3.get() > 1);
   }
 
-}
+
+
+  public void testRetry() throws Throwable {
```

Comment:
there should be one test method per test case imo

Reply:
in mine too, but that's how they are in the solr project. This is a copy paste of another test with a few modifications